taskvine/src/manager/vine_manager.c

```diff
@@ -590,6 +591,13 @@ static vine_result_code_t get_completion_result(struct vine_manager *q, struct v
 			}
 		}
 
+		/* Update category disk info */
```

Comment:
So I think you have the right idea here of tracking the max disk space used.

But isn't that the purpose of `category->max_resource_use->disk` ?

(And if not, why not?)